src/pages/experiments/[experimentId]/plots-and-tables/normalized-matrix/index.jsx

```diff
@@ -91,6 +94,14 @@ const NormalizedMatrixPage = (props) => {
     setMetadataCellSets(metadataTracks.map((track) => track.children).flat());
   }, [metadataTracks, cellSets.accessible]);
 
+  // TODO, is this needed?
```

Comment:
Which part? if you are asking about `useConditionalEffect` vs `useEffect`.

`useConditionalEffect` is different from `useEffect` in that it runs only when its dependenices `[CLMTracks, CLMPerSampleTracks, cellSets.accessible]` changed.

`useEffect` runs every time any of its dependencies was set again, this means if they were set again but their values didn't change it would run again anyways.